blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeImpl.java

```diff
@@ -96,6 +99,22 @@ public String getStartTimeString(){
         return edges;
     }
 
+    @Override
+    public Collection<BlueDownstreamBuild> getDownstreamBuilds() {
```

Comment:
Don't see why we need additional method, we already have `getActions()`, we returned filtered actions. You do not need `BlueDownstreamBuild` as well if you can construct `NodeDownstreamBuildAction` with link.